.github/workflows/CI.yml

```diff
@@ -40,11 +40,8 @@ jobs:
       run: ./build ci
       if: ${{ runner.os != 'Windows' }}
 
-    - name: Test - Persistence
-      run: dotnet bin/Debug/Persistence.Tests/Persistence.Tests.dll
-
-    - name: Test - PAModel
-      run: dotnet bin/Debug/PAModelTests/PAModelTests.dll
+    - name: Test
+      run: dotnet test src/PASopa.sln --no-build
```

Comment:
For maintainers: I would recommend to enable TRX report and Code Coverage.

Reply:
go ahead and enable if you would like.

Reply:
Will follow-up on this as it wasn't already enabled previously.